src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -1389,8 +1390,8 @@ function getEditorsContext(accessor: ServicesAccessor, resourceOrContext?: URI |
 	};
 }
 
-export function getEditorsFromContext(accessor: ServicesAccessor, resourceOrContext?: URI | IEditorCommandsContext, context?: IEditorCommandsContext): { editor: EditorInput; group: IEditorGroup }[] {
-	const { editors, groups } = getEditorsContext(accessor, resourceOrContext, context);
+export function resolveEditorsContext(context: EditorsContext): { editor: EditorInput; group: IEditorGroup }[] {
```

Comment:
We have a type that can be used which would make this more consistent: `IEditorIdentifier`

Reply:
`IEditorIdentifier` is not a good match here. It does not make sense to return the `groupId` if I already have the `group` instance and the consumer will require the `group` instance anyways.
See `resolveCommandsContext` which already existed and also does it this way. I think we should leave this. Maybe we can clean up this entire file during debt week.